submissions/fabunmiTolulope/css/product-page.css

```diff
@@ -0,0 +1,437 @@
+*{
+	box-sizing: border-box;
+	margin: auto;
+}
+body{
+	font-size: 16px;
+	font-family: "Georgia", Times New Roman;
+}
+header{
+	flex: 1 100%;
+	background-color: #B7EEB7;
+	height: 70px;
```

Comment:
Use padding instead of height @ToluEnny24

Reply:
I have changed it but will require an explanation to why padding

Reply:
The pading worked for the web view not the mobile view and the tab, and for that i used vh not px